src/main/scala/cromwell/binding/IOInterface.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.binding
```

Comment:
Hmm it could be in engine too I guess yes, it made sense to me to put it in binding because it's supposed to give the Wdl Library function (which are in bindings) a way to do IO, and the implementation would be backend dependant. But I don't mind moving it to engine.